Lib/test/test_tix.py

```diff
@@ -2,14 +2,18 @@
 from test import support
 from test.support import import_helper
 import sys
+import warnings
 
 # Skip this test if the _tkinter module wasn't built.
 _tkinter = import_helper.import_module('_tkinter')
 
 # Skip test if tk cannot be initialized.
 support.requires('gui')
 
-from tkinter import tix, TclError
+warnings.simplefilter('ignore', DeprecationWarning)
+from tkinter import tix
+warnings.simplefilter('always', DeprecationWarning)
```

Comment:
Instead of messing with global filters, you can use `tix = support.import_module('tkinter.tix', deprecated=True)`.